### ISSUE TITLE
Beta channel doesn't find stable releases any longer

### ISSUE THREAD

**Issue by tooomm** [Cockatrice/Cockatrice#3135]
Starting with the update to 2.5.0 - released on 2nd march - (from a former 2.5.0-beta) via the build in updater, I realized that the "Beta Channel" no longer finds and suggests updates to full stable versions of the app:
![beta channel](https://user-images.githubusercontent.com/9874850/37425960-7ac791c8-27c5-11e8-9de3-4d158d5c5d60.png)

I'm pretty sure this used to be different. It makes no sense to only look for releases tagged with "pre-releases".

Users should get any most recent update, including normal releases, if opting to receive beta releases additionally. Otherwise there will be a weird delay and gap.

**Comment by Daenyth** [comment]
I think this may have come in where we fixed the bug where installing a beta would then try to auto update to an older full release because it lost the beta channel setting.

**Comment by tooomm** [comment]
Hmm, I can't remember such a change/fix. But I wasn't aware of lost channel settings either... :)

A quick search didn't reveal a PR?! Can you link me, please @Daenyth?

**Comment by Daenyth** [comment]
I'm not sure now

**Comment by ZeldaZach** [comment]
I'm guessing this came about from when we changed how line endings worked for the updater and removed the devUpdates field

**Comment by ctrlaltca** [comment]
looking at the "BETARELEASE_URL" https://api.github.com/repos/Cockatrice/Cockatrice/releases it looks that:
 * version Development-2.5.0-beta.16 is dated: ``` "published_at": "2018-02-19T08:01:27Z", ```
 * while the stable Release-2.5.0 is dated: ``` "published_at": "2018-03-02T21:51:53Z" ```

Still, the github API page is showing the beta release first, so cockatrice is assuming it's newer and picking it up as the latest available release.
I don't know why the github api is not showing releases sorted by date; I've opened a forum discussion here hoping for a solution: https://github.community/t5/How-to-use-Git-and-GitHub/Releases-API-can-t-figure-out-order/m-p/5037#M1640

**Comment by ctrlaltca** [comment]
Turns out releases are sorted first by "prerelease" status and then by "published_at" date.
We didn't notice this in the past because we only listed stable releases and "beta" prereleases only if they are more recent than the latest stable.
Now in the releases page there are still some betas that are older than the last stable, and the problem got noticed.
We could workaround this by now by deleting all the previous beta when a stable gets released.
